README.md

```diff
@@ -87,9 +87,10 @@ During application boot, you can call ```hystrix.ConfigureCommand()``` to tweak
 
 ```go
 hystrix.ConfigureCommand("my_command", hystrix.CommandConfig{
-	Timeout:               1000,
-	MaxConcurrentRequests: 100,
-	ErrorPercentThreshold: 25,
+	Timeout:                     1000,
+	MaxConcurrentRequests:       100,
+	ErrorPercentThreshold:       25,
+	QueueSizeRejectionThreshold: 100,
```

Comment:
How would you pick a good value for `QueueSizeRejectionThreshold`?

Reply:
The default value of `QueueSizeRejectionThreshold` is currently equal to the `MaxConcurrentRequests`. This should take care of request spike which is 2x the usual.
2x seems to be a decent number, although in some of the use-cases I have seen 3x-4x as well (in my workplace).
Netflix also use a number equal to `MaxConcurrentRequests`, although I am open to suggestion for a better default value.